src/components/AssigneesContainer/AssigneesContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import { Menu, Input, ItemImage, Segment } from 'semantic-ui-react'
+
+const AssigneesContainer = () => {
```

Comment:
I think, to avoid any confusion we should use the term Container only for ReduxContainer.

Reply:
I renamed them as AssigneesSegment